datafusion/core/tests/sql/group_by.rs

```diff
@@ -70,6 +70,45 @@ async fn group_by_date_trunc() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn distinct_group_by_limit() -> Result<()> {
```

Comment:
Does this test add additional coverage compared to the tests in `datafusion/sqllogictest/test_files/aggregate.slt`?

Reply:
Not really, except that it uses `mode=Single`.  Removed the test.